### PR TITLE
allwinner: extend support for H5 CPU (sun50i-h5)

### LINKED ISSUE #21: allwinner: extend support for H5 (sun50i-h5)
**What kind of new feature are you looking for?**
(Keep the one that applies, please describe)
- Hardware: Allwinner H5, NanoPi NEO Core2
- Software: new CPU support

**Do you plan to:**
- Contribute an initial driver: Yes
- Write unit tests: No
- Update https://github.com/periph/cmd to use the new functionality: No

### PR DESCRIPTION
**Pull request by TzeWey**
Fixes #21

Tested on a custom target but not all IOs.
Verified GPIO Port A and Port C1 are working.

### CODE REVIEW

File: allwinner/h5.go [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+	"periph.io/x/conn/v3/pin"
+	"periph.io/x/host/v3/sysfs"
+)
+
+// mappingH5 describes the mapping of the H5 processor gpios to their
+// alternate functions.
+//
+// It omits the in & out functions which are available on all gpio.
+//
+// The mapping comes from the datasheet page 23:
```
Comment (maruel) [inline allwinner/h5.go:22]: Page 23?

**Review by maruel — COMMENTED** [review]
Thanks a lot!!

Only one small nit.

File: allwinner/h5.go (hunk 1 above)
Comment (TzeWey) [reply allwinner/h5.go:22]: Good catch, I forgot to update the page number.

Comment (TzeWey) [reply allwinner/h5.go]: Updated in latest commit

**Comment by maruel** [comment]
Thanks!

**Pull request merged**